src/test/java/seedu/storemando/logic/parser/ClearCommandParserTest.java

```diff
@@ -0,0 +1,36 @@
+package seedu.storemando.logic.parser;
+
+import static seedu.storemando.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.storemando.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.storemando.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.storemando.logic.commands.ClearCommand;
+import seedu.storemando.model.item.LocationContainsPredicate;
+
+public class ClearCommandParserTest {
+
+    private final ClearCommandParser parser = new ClearCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
```

Comment:
This method name states that parsing an empty argument would throw parse exception but you are testing for parse success. Perhaps, you might want to change the method name.

Reply:
Noted, I have changed the method name to parse_emptyArg_returnsClearCommand()